### PR TITLE
Use session cookies and store filter settings in sessionStorage

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
Tested with Chrome and FF. Please check [this](https://docs.djangoproject.com/en/1.10/ref/settings/#std:setting-CSRF_COOKIE_AGE) whether the described effects with the bookmark are such a negative thing. I can not confirm this. (Info: if the csrf cookie is wrong, you will just get a 403 - forbidden; clearing the cookie helps).

### CODE REVIEW

File: openslides/core/static/js/core/site.js [hunk 1]
```diff
@@ -343,6 +343,13 @@ angular.module('OpenSlidesApp.core.site', [
     }
 ])
 
+.config([
+    '$sessionStorageProvider',
+    function ($sessionStorageProvider) {
+        $sessionStorageProvider.setKeyPrefix('OS-');
```
Comment (normanjaeckel) [inline openslides/core/static/js/core/site.js]: Can we use a more explicit prefix here like `OpenSlides-`?

Comment (FinnStutzenstein) [inline openslides/core/static/js/core/site.js]: Ok, should the `csrftoken` also be renamed to `OpenSlidesCsrfToken` to keep every cookie and storage with the same prefix?

Comment (normanjaeckel) [inline openslides/core/static/js/core/site.js]: Ok, why not.

**Comment by emanuelschuetze** [comment]
@normanjaeckel Please review again.
@FinnStutzenstein Do you still want to rename the csrftoken cookie?

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze yes, I'll do this tomorrow morning, so this pr can be merged tomorrow.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Ok.

**Pull request merged**